### PR TITLE
Bump odo version in installation docs

### PR DESCRIPTION
**Pull request by valaparthvi**
**What type of PR is this:**
/kind documentation

**What does this PR do / why we need it:**
This PR updates the version number in installation docs.
**Which issue(s) this PR fixes:**

Fixes #

**PR acceptance criteria:**

- [ ] Unit test 

- [ ] Integration test 

- [x] Documentation 

**How to test changes / Special notes to the reviewer:**

### CODE REVIEW

**Review by rm3l — COMMENTED** [review]
Should we change the version in the top navbar dropdown menu as well? It still shows Alpha 2.
cc @cdrage 

image

**Comment by cdrage** [comment]
> 

That can be found in `docusaurus.config.js` that you can change for that value.

And yup! We should change the link as well with regards to latest, etc.

**Comment by rm3l** [comment]
/hold

We should wait for #5909 to be merged first, then also update the `odo.version` property for the Maven Plugin installation method.

**Comment by valaparthvi** [comment]
> Should we change the version in the top navbar dropdown menu as well? It still shows Alpha 2. cc @cdrage
> 
> image

If we change this, I am inclined to change all the other references of 2.5.0 which includes renaming versioned-docs and all of its references. Wouldn't it be better to take that up in a separate PR?

**Comment by cdrage** [comment]
> > Should we change the version in the top navbar dropdown menu as well? It still shows Alpha 2. cc @cdrage
> > image
> 
> If we change this, I am inclined to change all the other references of 2.5.0 which includes renaming versioned-docs and all of its references. Wouldn't it be better to take that up in a separate PR?

Just change the Alpha 2 to Beta 1 in the docusaurus.config.js 

We can change the references of 2.5.0 in another PR.

File: docs/website/docs/overview/installation.md [hunk 1]
```diff
@@ -31,12 +31,12 @@ Installing `odo` on `amd64` architecture:
 
 1. Download the latest release from the mirror:
 ```shell
-curl -L https://developers.redhat.com/content-gateway/rest/mirror/pub/openshift-v4/clients/odo/v3.0.0~alpha2/odo-linux-amd64 -o odo
+curl -L https://developers.redhat.com/content-gateway/rest/mirror/pub/openshift-v4/clients/odo/latest/odo-linux-amd64 -o odo
```
Comment (rm3l) [inline docs/website/docs/overview/installation.md:34]: We should use `v3.0.0~beta1` everywhere in this file, as the "latest" link will be fixed soon to point to `v2.5.1` (latest stable release).

**Review by rm3l — CHANGES_REQUESTED** [review]
A few comments below.

Please also consider updating the version mentioned in the "Installing `odo` using Maven plugin" section (introduced in https://github.com/redhat-developer/odo/pull/5909)

File: docs/website/docs/overview/installation.md (hunk 1 above)
Comment (valaparthvi) [reply docs/website/docs/overview/installation.md:34]: >We should use v3.0.0~beta1 everywhere in this file, as the "latest" link will be fixed soon to point to v2.5.1 (latest stable release).

Do we have an issue open to track this change?

Comment (rm3l) [reply docs/website/docs/overview/installation.md]: This is an issue for the EXD team - they need to update the link manually. They had created an issue on the Content Gateway to have this done automatically, but this still does not work as expected. See [https://issues.redhat.com/browse/CGW-844](https://issues.redhat.com/browse/CGW-844)

File: docs/website/docs/overview/installation.md [hunk 2]
```diff
@@ -31,12 +31,12 @@ Installing `odo` on `amd64` architecture:
 
 1. Download the latest release from the mirror:
 ```shell
-curl -L https://developers.redhat.com/content-gateway/rest/mirror/pub/openshift-v4/clients/odo/v3.0.0~alpha2/odo-linux-amd64 -o odo
+curl -L https://developers.redhat.com/content-gateway/rest/mirror/pub/openshift-v4/clients/odov3.0.0~beta1odo-linux-amd64 -o odo
```
Comment (rm3l) [inline docs/website/docs/overview/installation.md:34]: Missing `/` between `~beta1` and `odo-linux` here and below ;-)

**Review by rm3l — CHANGES_REQUESTED** [review]

File: docs/website/docs/overview/installation.md (hunk 2 above)
Comment (rm3l) [reply docs/website/docs/overview/installation.md:34]: Ah, and also between `odo` and `v3.0.0~beta1`

Comment (valaparthvi) [reply docs/website/docs/overview/installation.md]: :facepalm: Fixed it, thanks for catching!

**Comment by dharmit** [comment]
> We should use `v3.0.0~beta1` everywhere in this file, as the "latest" link will be fixed soon to point to `v2.5.1` (latest stable release).

Yikes! I understand the reasons, but it sucks. 😞 

/lgtm
/override Kubernetes Integration Tests/Kubernetes Integration Tests

**Pull request merged**